### PR TITLE
example diagrams

### PR DESCRIPTION
**Pull request by maxzhen**

### CODE REVIEW

File: src/Documentation/fpga/xrt.rst [hunk 1]
```diff
@@ -596,6 +596,72 @@ The driver object model looks like the following::
...
+      v           v             v         v          v          v
+ +------------+  +------+    +------+ +------+    +-----+ +-----------+
+ | xmgmt_main |  | VSEC |    | GPIO | | QSPI |    | CMC | | AXI-GATE0 |
+ +------------+  +------+    +------+ +------+    +-----+ +-----------+
+                             +---------+          +------+ +-------+
+                             | MAILBOX |          | ICAP | | CALIB |
+                             +---------+          +------+ +-------+
+                                                  +-----------+
+                                                  | AXI-GATE1 |
+                                                  +-----------+
```
Comment (sonals) [inline src/Documentation/fpga/xrt.rst:624]: Looks good. Wondering if we should draw arrow from group1 to MAILBOX and group2 to ICAP, CALIB, AXI-GATE1?

File: src/Documentation/fpga/xrt.rst [hunk 2]
```diff
@@ -596,6 +596,72 @@ The driver object model looks like the following::
...
+     +----------------+---+-----------+--------------+
+     |                |               |              |
+     v                v               v              v
+ +--------+      +--------+      +--------+       +-----+
+ | CLOCK0 |      | CLOCK1 |      | CLOCK2 |       | UCS |
+ +--------+      +--------+      +--------+       +-----+
+ +-------------+ +-------------+ +-------------+
+ | CLOCK-FREQ0 | | CLOCK-FREQ1 | | CLOCK-FREQ2 |
+ +-------------+ +-------------+ +-------------+
+
```
Comment (sonals) [inline src/Documentation/fpga/xrt.rst:664]: Same as before. But we should move group3 after group2 so it looks chronological?

File: src/Documentation/fpga/xrt.rst (hunk 1 above)
Comment (maxzhen) [reply src/Documentation/fpga/xrt.rst]: fixed

File: src/Documentation/fpga/xrt.rst (hunk 2 above)
Comment (maxzhen) [reply src/Documentation/fpga/xrt.rst:666]: fixed

**Pull request merged**